packages/app/app/containers/MiniPlayerContainer/index.tsx

```diff
@@ -31,6 +45,14 @@ const MiniPlayerContainer: React.FC = () => {
         style={{
           zIndex: isMiniPlayerEnabled && 1000
         }}
+        lyricsComponent={
```

Comment:
Since you pass an element, not a component, rename this prop accordingly

Reply:
Done :)